modules/social_features/social_follow_taxonomy/modules/social_follow_tag/social_follow_tag.install

```diff
@@ -42,3 +43,35 @@ function social_follow_tag_update_8801() {
     }
   }
 }
+
+/**
+ * Update followtag block.
+ */
+function social_follow_tag_update_8802() {
+  $config_path = drupal_get_path('module', 'social_follow_tag') . '/config/optional';
```

Comment:
It's not a good idea to load these config files from optional / install (in the below update hook)
rather move a copy to /static which never changes, so you don't have any issues when these yml files change and the update hook might not have run